xapian-core/tests/harness/testsuite.cc

```diff
@@ -706,9 +711,20 @@ test_driver::do_run_tests(vector<string>::const_iterator b,
 		case PASS:
 		    ++res.succeeded;
 		    if (verbose || !use_cr) {
-			out << col_green << " ok" << col_reset << endl;
+			if (test_duration.count() >= 10000) {
```

Comment:
This is a little bit of a magic number, so it might be a good idea to drop a comment ahead of it making it clear that 10s is somewhat arbitrary, but "not too long and not too short". (There could be advantage in making it configurable, pulling from an environment variable with a default of 10s — it would mean we could keep an eye out for really long tests, and periodically go after medium-length ones. Not sure it's worth it right now, though, so probably better to keep a note at most.)

Reply:
It could at least be pulled out into a `SLOW_TEST_THRESHOLD` constant defined near the start of the source file - the name then documents what we're testing here, and someone looking to adjust it can find it more easily.

Reply:
Added a variable.